nemo/collections/asr/modules/transformer/bridge_encoders.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+from nemo.collections.asr.modules.transformer.transformer_encoders import TransformerEncoder
+from nemo.collections.asr.modules.transformer.transformer_modules import AttentionBridge
+
+__all__ = ["BridgeEncoder"]
+
+
```

Comment:
Im skipping review of the transformer stuff since its ported from NLP which I assume is tested.